_posts/2021-07-16-1.1.0-released.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Crystal 1.1.0 is released!
+summary: The first post-1.0 release
+thumbnail: +
+author: beta-ziliani
+release_notes: true
+---
+
+We are releasing the first post-1.0 release, making efforts to keep our promise of making regular releases each 3 months, a bit delayed because of the [conference](/conference), and with special focus on keeping the language stability. Below we list the most important or interesting changes, without mentioning the several bugfixes. For details visit the [release's notes](https://github.com/crystal-lang/crystal/releases/tag/1.1.0).
+
+## Stats
+
+In this release we included [166 PRs since the 1.0.0 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.1.0) by 28 contributors. We thank all the effort put into improving the language and its stdlib! ❤️
+
+## Language changes
+
+1. Tuple literals now allow the use of splats:
+
+```crystal
+t = {1, 'a'}
+t2 = {true, *t} # => {true, 1, 'a'}
+typeof(t2)      # => Tuple(Bool, Int32, Char)
+
+def f1(x : {Bool, *{Int32, Char}}); end
+def f2(x : Tuple(Bool, *{Int32, Char})); end
+```
+
+Note that it also works for specifying types.
```

Comment:
The changelog says:

> Support splat expansions inside tuple **and array literals**

So I think we should mention that, because it's really powerful!

Some things you can do now that you couldn't (easily) do before:

```crystal
r = 'a'..'e'

a = ['<', *r, '>'] # => ['<', 'a', 'b', 'c', 'd', 'e', '>']
```

Note that you can splat any enumerable inside literals, not only tuples!

Another example:

```crystal
a = 1..10
b = 5..15

Set{*a, *b} # => Set{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}
```

So it actually works for array-like types too!

Reply:
Another example:

```crystal
a = [1, 2, 3]
b = [0, *a, *a] # => [0, 1, 2, 3, 1, 2, 3]
```

Reply:
I added the examples. Thanks!